lte/protos/session_manager.proto

```diff
@@ -706,10 +722,43 @@ message QosRules {
    uint32 qos_flow_identifier = 7;
 }
 
+enum prem_capab {
+  SHALL_NOT_TRIGGER_PRE_EMPTION = 0;
+  MAY_TRIGGER_PER_EMPTION = 1;
+}
+enum  prem_vuner {
+  NOT_PREMTABLE = 0;
+  PRE_EMPTABLE = 1;
+}
+
+message allocation_and_retention_prio {
+   uint32       prio_level = 1;
+   prem_capab   pre_emtion_cap = 2;
+   prem_vuner   pre_emtion_vul = 3;
+}
+
+message qos_characteristics {
+   uint32 fiveQi = 1;
+}
+
+message flow_level_params {
+    qos_characteristics qos_chars = 1;
+    allocation_and_retention_prio alloc_reten_prio = 2;
+}
+message qos_flow {
+   uint32  qos_flow_ident = 1;
+   flow_level_params  param = 2;
+}
+message  qos_flow_request_list {
+   qos_flow flow  = 1;
+}
+
+
```

Comment:
Reuse FlowQoS definition from policydb.proto

Reply:
All required qos default/mandatory params required by gNodeB are not matching existing Qos definitions which are available from "policydb.proto",  more than that AMF can't generate/add any new Qos Parameters by itself, as it relays whatever SMF sends.

Reply:
Reuse FlowQoS definition, done the code changes from SMF side

Reply:
@gurrapualt I thought these were clarified in our conversation. All of these are already defined in the existing protos; which one is not defined?